packages/sit-onyx/src/i18n/locales/de-DE.json

```diff
@@ -1,16 +1,33 @@
 {
   "optional": "(optional)",
   "validations": {
```

Comment:
What do you think about grouping them by validation type instead? So something like;

```json
  "validations": {
    "badInput": {
      "preview": "Invalid input",
      "long": "\"{value}\" does not match the expected type."
    }
  },
```

Imho its easier to compare them or add new validations in the future